builder/publish.go

```diff
@@ -51,47 +77,88 @@ func PublishImage(image string, handler string, functionName string, language st
 			return nil
 		}
 
-		buildOptPackages, buildPackageErr := getBuildOptionPackages(buildOptions, language, langTemplate.BuildOptions)
+		if remoteBuilder != "" {
+			logger := log.New(os.Stderr, "", log.LstdFlags)
+			logger.SetFlags(log.Ldate | log.Ltime | log.LUTC)
 
-		if buildPackageErr != nil {
-			return buildPackageErr
+			tempDir, err := os.MkdirTemp(os.TempDir(), "builder-*")
+			if err != nil {
+				logger.Printf("%s %s", functionName, err)
```

Comment:
This may cause a double log.

Reply:
`fmt.Errorf("failed to create temporary directory for %s, error: %w", functionName, err)`